app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -67,26 +67,35 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <androidx.recyclerview.widget.RecyclerView
-      android:id="@+id/submitted_answer_recycler_view"
+    <LinearLayout
```

Comment:
I have added `LinearLayout` just to make sure that `contentDescription` works as it does not work when directly added to `RecyclerView`.

Reply:
ok, something new.

Reply:
Was stuck on this for hours.